contracts/root/lib/ChildChainVerifier.sol

```diff
@@ -1,23 +1,24 @@
 pragma solidity ^0.5.2;
 
-import { RLPReader } from "solidity-rlp/contracts/RLPReader.sol";
 import { BytesLib } from "../../common/lib/BytesLib.sol";
 import { Common } from "../../common/lib/Common.sol";
-import { RLPEncode } from "../../common/lib/RLPEncode.sol";
-import { Common } from "../../common/lib/Common.sol";
 import { MerklePatriciaProof } from "../../common/lib/MerklePatriciaProof.sol";
+import { RLPEncode } from "../../common/lib/RLPEncode.sol";
+import { RLPReader } from "solidity-rlp/contracts/RLPReader.sol";
 
 import { Registry } from "../../common/Registry.sol";
 
 library ChildChainVerifier {
   using RLPReader for bytes;
   using RLPReader for RLPReader.RLPItem;
 
+  event YOYO(address rootToken, address childToken);
+
   function processBurnReceipt(
     bytes memory receiptBytes, bytes memory path, bytes memory receiptProof,
     bytes32 receiptRoot, address sender, Registry registry)
     internal
-    view
+    // view
```

Comment:
why make it internal? IMO keeping it `public view` is good so that other people can just verify data easily.

Reply:
bro I made it public and migrations started failing 😆

Reply:
why tho?